### ISSUE TITLE
Screen reader is unable to focus on "Personalization Settings" modal

### ISSUE THREAD

**Issue by Sanjog1605** [benetech/MathShare#1298]
# Description
When the  'Personalization' modal/dialog opens, the screen reader is unable to bring it's focus on to the dialog.

## Steps to reproduce the behavior:
1. Go to [Mathshare - QA Webpage](https://mathshare-qa.diagramcenter.org/#/app)
2. Login using 'Google' or 'Microsoft' accounts by pressing  the "Login" button.
3. Click on User Profile button to expand the menu.
4. Scroll down to 'Personalization' button by pressing the down arrow.
5. Error: Screen reader does not read the 'Personalization Settings' dialog box.

## Expected Result
Screen reader should read the contents of the modal and the focus should remain inside the modal.

![Personalization Settings Modal](https://user-images.githubusercontent.com/68078108/87422045-8f4bf280-c5f5-11ea-9b50-33ca007ecceb.png)

### Desktop 
 - OS: [Windows 10 Pro v1909]
 - Browser: Google Chrome (v83.0.4103.116) and Mozilla Firefox (v78.0.2)
 - Screen Reader: NVDA 2020.1

## Additional Comments
I encountered the above mentioned issue while I was researching about #1006 as I was unable to access the interactive elements within the model. (@alexrcabral )

**Comment by johnhbenetech** [comment]
Hi @Sanjog1605 , can you recheck this. We made a recent update to modals which hopefully should fix this

**Comment by Sanjog1605** [comment]
Hello @johnhbenetech  Sir, I have checked it and it is working fine. I gave the comment under the issue #814.

**Comment by bognasoldev89** [comment]
**Tested with success**

**Environment**
- QA env
- Windows 10
- Browsers
-- Firefox 78.0.2
-- Chrome 84.0.4147.89
- NVDA 2020.1

**Verified cases**
- Opening personalization settings with the screen reader

**Summary**
I opened this modal with NVDA screen reader and it was read aloud.
